engine/reference/commandline/scout_compare.md

```diff
@@ -10,4 +10,7 @@ in the source repository on GitHub:
 
 https://github.com/docker/scout-cli-plugin
 -->
+
+{% include scout-early-access.md %}
+
```

Comment:
@thaJeztah did you see the comment above this saying this file is auto generated? Just checking this isn't going to get removed automatically at some point in the future.

Reply:
@mikeparker ah, yes, I guess the comment is slightly ambiguous on these; so the markdown "stub" files themselves are not automatically generated, but the content is generated from YAML files (https://github.com/docker/docs/tree/c0402932ca01a11eb663080191ba5f4b7e9194dd/_data/scout-cli) combined with a template.

So unless scout also generates the stubs, I think we're good 👍